utils/config/config.go

```diff
@@ -329,6 +332,11 @@ func convertIfNeeded(content []byte) ([]byte, error) {
 		}
 		fallthrough
 	case "1":
+		log.Debug("Converting JFrog config to the latest version...")
```

Comment:
I suggest changig this to:
log.Info("Converting JFrog CLI's config to the latest version...")